src/common/RAJAPerfSuite.hpp

```diff
@@ -82,7 +86,9 @@ enum KernelID {
   Basic_PI_REDUCE,
   Basic_REDUCE3_INT,
   Basic_TRAP_INT,
-
+#ifdef RUN_KOKKOS // move this up to the point implemented with Kokkos
```

Comment:
Not sure why this is guarded. But, it looks to me like it will break the code if it is.

Reply:
The reason that happens is because the Kokkos variants are not implemented for the kernels coming after this point, and your kernel loop count comes from this enum (NumKernels). Eventually when all kernels have Kokkos variants this ifdef and the one after is not needed anymore. I.e. every subsequent PR which introduces the next set of kernels will move this further down until we are converged.

Reply:
See comment about defaulting runKokkosVariant to avoid ifdefing.